test/unit/streaming.net.HTTPLoader.js

```diff
@@ -9,6 +9,7 @@ import {
 }
     from '../../src/streaming/vo/metrics/HTTPRequest';
 import Settings from '../../src/core/Settings';
+import { sleep } from './helpers/Async';
 
 const expect = require('chai').expect;
```

Comment:
A test should be added to verify that an optional modifyRequestURL function is supported. Once this ships and people are supplying a requestModifer without this method it shouldn't regress.